test/cmd/authentication.sh

```diff
@@ -41,9 +41,10 @@ users:
 - name: invalid_token_user
   user:
     exec:
-      apiVersion: client.authentication.k8s.io/v1beta1
+      apiVersion: client.authentication.k8s.io/v1
       # Any invalid exec credential plugin will do to demonstrate
```

Comment:
I would copy one of these tests and leave it as beta so we continue to have coverage.

Reply:
+1 for leaving existing tests intact and adding a v1 test

Reply:
https://github.com/kubernetes/kubernetes/pull/102890/commits/5a3139890ba7da6488233cd389af371b3d77363a runs all basic tests with v1 and v1beta1

https://github.com/kubernetes/kubernetes/pull/102890/commits/c1e9e36256f871cd811706552f919b8b4f9c1c09 adds a negative v1 test